ament_index_python/README.md

```diff
@@ -0,0 +1,5 @@
+# ament_index_python
+
```

Comment:
I'd add at least a one sentence summary, maybe just the description from the package.xml. You might also want to link to the overview of the ament resource index.

https://github.com/ament/ament_cmake/blob/master/ament_cmake_core/doc/resource_index.md